rfcs/proposed/simplified_numa_support/README.md

```diff
@@ -0,0 +1,179 @@
+# Simplified NUMA support in oneTBB
```

Comment:
I would probably call it "Improved NUMA support".
Correspondingly, the RFC folder could be `numa_support_improvements`, meaning that NUMA support is a core feature and improvements are the gist of the proposal.

Reply:
As far as I understand the whole `numa_support_improvements` or `simplified_numa_support` directory will be moved to `rfcs/supported` directory once these improvements are accepted. There may be another set of NUMA improvements in the future which could result in another `numa_support_improvements` directory be created in the same place. And then, when this new set is again accepted, it moves to the same directory. I see potential naming clash issue... It is not related to naming of this directory, but to the naming approach in general. Surely, we could use `numa_support_improvements2` as the name of the new directory, but I believe we can do better from the very beginning.

I propose having the directory with the name related to the feature itself, e.g., `numa_support`, without additionals such as `simplified` or `improvement`. This way we will convey the idea that the documents inside directly affect the support of a particular feature. For resolving naming clashes I propose having the file to be named as precise as possible to what the proposal changes avoiding general terms/adjectives such as `improved`, `increased`, etc.. For example, for sub-RFC that I wrote, I suggest naming the file to something like `introduce_tbbbind_static_library` or `introduce_tbbbind_statically_linked_with_hwloc`; for NUMA-aware allocators name something like `introduce_numa-aware_allocator`; for task_group dependencies name something like `introduce_dependencies_for_tasks_in_task_group`; and so on. This way we would avoid name clashing and still this allows grouping similar rfcs together into dedicated folder such as `numa_support`. Otherwise, I am afraid that the feature is not elaborated enough to be proposed since it sounds too generic in our mind.

Reply:
I will move to a numa_support directory and change title to "NUMA support"